Directory.Build.props

```diff
@@ -3,7 +3,6 @@
 
   <PropertyGroup>
     <Deterministic>true</Deterministic>
-    <TargetFrameworks>netstandard2.0;netstandard2.1;net472</TargetFrameworks>
```

Comment:
This shouldn't be in the `Directory.Build.props` as the library and 'tests' project do target completely different frameworks. I noticed you tweaking this in your PR.